client/src/holdings/AddDividendDialog.tsx

```diff
@@ -93,6 +94,7 @@ export function AddDividendDialog({holding, onDividendCreated}: IProps) {
                   Add
                 </Button>
               </DialogActions>
+              <div>{Boolean(isSubmitting) && <CircularProgress />}</div>
```

Comment:
I think it looks better if it's the first element after `<DialogActions>`, then it fits nicely with the buttons and it's clear that it related to the one the user has just clicked:

```tsx
{!!isSubmitting && (
  <CircularProgress size={25} style={{marginRight: '10px'}} />
)}
```

(The !! is another way of writing Boolean)

Then you can do the same for the AddTradeDialog too